### ISSUE TITLE
No completions on imported functions

### ISSUE THREAD

**Issue by Astrantia** [Microsoft/python-language-server#785]
```py
from selenium import webdriver
webdriver.Chrome(options=options).find_elements_by_css_selector().
```
This will not show any completions in vscode

![image](https://user-images.githubusercontent.com/32885386/54566087-43a81b00-49a6-11e9-95a8-f60b90e82c53.png)

**Comment by jakebailey** [comment]
Are you using v0.2? If not, please retest with the beta build by adding this to your VS Code settings and restarting:

```
"python.analysis.downloadChannel": "beta"
```

**Comment by Astrantia** [comment]
I am

**Comment by Astrantia** [comment]
@jakebailey were you able to reproduce this?

**Comment by jakebailey** [comment]
Yes. This one is tough; these functions call `find_element`, which calls `execute` and grabs some value from the response, which comes from `_unwrap_value`, which is complicated.

**Comment by Astrantia** [comment]
That means there are issues with 'tough' functions like that not getting completions. This should be given a priority 😃

**Comment by MikhailArkhipov** [comment]
Not necessarily. Depends on the how popular is the library and frequency of use of particular function.

Besides, not all functions can be analyzed with the definitive result. In dynamic untyped languages there is often not enough information in the code to tell what particular method returns. Consider `void* func(void*)` in C++ or `object func(object)` in C#. In this case stubs and annotations are source of information or the actual function body eval. However, evaluation also does not always work, esp if code itself creates objects dynamically or even compiles code written as strings.

**Comment by Astrantia** [comment]
Are you saying that PyCharm has a hardcoded fix for this?

**Comment by MikhailArkhipov** [comment]
It is a possibility, special casing is not unusual. PyCharm also has a lot of custom stubs.

**Comment by roman-geraskin** [comment]
Is there any workaround? VS code completions are not usable with selenium completely.

![image](https://user-images.githubusercontent.com/51863158/71437002-c19d0d80-2700-11ea-8a38-e367d8ffc86b.png)
![image](https://user-images.githubusercontent.com/51863158/71437112-4556fa00-2701-11ea-8c3f-9945c197771a.png)

Version: 1.41.1 (system setup)
Commit: 26076a4de974ead31f97692a0d32f90d735645c0
Date: 2019-12-18T14:58:56.166Z
Electron: 6.1.5
Chrome: 76.0.3809.146
Node.js: 12.4.0
V8: 7.6.303.31-electron.0
OS: Windows_NT x64 10.0.17763

Name: Python
Id: ms-python.python
Description: Linting, Debugging (multi-threaded, remote), Intellisense, code formatting, refactoring, unit tests, snippets, and more.
Version: 2019.11.50794
Publisher: Microsoft

Microsoft Python Language Server version 0.5.10.0
